lib/cluster.js

```diff
@@ -684,10 +684,13 @@ function workerInit() {
 
   Worker.prototype.destroy = function() {
     this.exitedAfterDisconnect = true;
-    if (!this.isConnected()) process.exit(0);
-    var exit = process.exit.bind(null, 0);
-    send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-    process.once('disconnect', exit);
+    var code = 0;
```

Comment:
@cjihrig I choose to use your suggestion 1 because of the consistence with other functions in this module.